packages/lua/src/utils/printJSON.lua

```diff
@@ -0,0 +1,86 @@
+--- Calculates if the real length of a table is zero. This is used to check for empty dictionaries and arrays.
+---@param tbl table
+---@return boolean
+local function isEmpty(tbl)
+	local length = 0
+	for _, _ in pairs(tbl) do
+		length = length + 1
+	end
+
+	return length == 0
+end
+
+local function isDictionary(tbl)
```

Comment:
Depending on your usecase this may be fine for the empty table case but you can do
```lua
local function isDictionary(tbl)
    return next(tbl, 0) == nil and next(tbl) ~= nil
end
```
`next(tbl, 0)` is what ipairs does while `next(tbl)` is what pairs does